ledger_device_sdk/src/ecc.rs

```diff
@@ -530,6 +556,7 @@ impl_curve!(BrainpoolP512R1, 64, 'W');
 impl_curve!(BrainpoolP512T1, 64, 'W');
 impl_curve!(Stark256, 32, 'W');
 impl_curve!(Ed25519, 32, 'E');
+impl_curve!(Bls12381G1, 32, 'E'); // TODO: review type of curve (...and sigining impl)
```

Comment:
I guess we should not do this and expose the usual signing APIs yet. EdDSA or ECDSA are typically not used on BLS12381G1. We need to find a way to expose signing on BLS12381G2 though, which is a BLS signature, not ECDSA or EdDSA, and whose keys must be derived with eip2333 using G1

Reply:
Agreed, apart from impl. details w.r.t. being able to use the `impl_curve!` or not that I'll try to work out, I've found there is `ox_bls12381_sign`, is that supposed to be good enough for BLS12381G2 sig?